client/landing/gutenboarding/onboarding-block/create-site/index.tsx

```diff
@@ -1,60 +1,59 @@
 
 /**
  * Internal dependencies
  */
-import AnimatedPlaceholder from '../animated-placeholder';
 import CreateAndRedirect from './create-and-redirect';
 import { useNewQueryParam } from '../../path';
 import './style.scss';
+/**
+ * Internal dependencies
+ */
+import { ProgressBar } from '@automattic/components';
 
 /* eslint-disable wpcalypso/jsx-classname-namespace */
 const CreateSite: FunctionComponent< {} > = () => {
 	const { __: NO__ } = useI18n();
 	const shouldTriggerCreate = useNewQueryParam();
+	const progressTotal = 100;
+	const progressStepCount = 3;
+	const [ progressComplete, setProgressComplete ] = React.useState( 0 );
+	const [ shouldCreateAndRedirect, setCreateAndRedirect ] = React.useState( false );
 
-	const createAndRedirect = shouldTriggerCreate ? <CreateAndRedirect /> : null;
+	useEffect( () => {
+		if ( progressComplete < progressTotal ) {
+			setTimeout( () => {
+				const progressFactor = progressTotal / progressStepCount;
+				setProgressComplete( progressComplete + progressFactor );
+			}, 2000 );
+		}
+
+		if ( progressComplete >= progressTotal ) {
+			setCreateAndRedirect( true );
+		}
+	}, [ progressComplete, shouldTriggerCreate ] );
 
 	return (
 		<div className="create-site__background">
-			{ createAndRedirect }
+			{ shouldCreateAndRedirect ? <CreateAndRedirect /> : null }
 			<div className="create-site__layout">
 				<div className="create-site__header">
-					<div className="create-site__toolbar">
-						<div className="create-site__placeholder create-site__placeholder-site">
-							Placeholder
-						</div>
-					</div>
-					<div className="create-site__settings">
-						<div className="create-site__placeholder create-site__placeholder-button">
-							Placeholder
-						</div>
-						<div className="create-site__placeholder create-site__placeholder-button">
-							Placeholder
-						</div>
-						<div className="create-site__placeholder create-site__placeholder-button">
-							Placeholder
-						</div>
+					<div className="gutenboarding__header-wp-logo">
+						<Icon icon="wordpress-alt" size={ 24 } />
 					</div>
 				</div>
 				<div className="create-site__content">
-					<div className="create-site__placeholder create-site__placeholder-title">Placeholder</div>
-					<div className="create-site__text">
-						<AnimatedPlaceholder
-							isSlow
-							texts={ [
-								NO__( 'We are creating your site.' ),
-								NO__( 'It will be ready in a moment.' ),
-								NO__( 'Almost there, hang on!' ),
-								NO__( 'Your site is almost ready!' ),
-								NO__( 'We are about to finish!' ),
-							] }
-						/>
+					<h1 className="create-site__title">{ NO__( 'Building your site' ) }</h1>
+					<div className="create-site__progress">
+						<ProgressBar value={ progressComplete } total={ progressTotal } />
 					</div>
+					{ /* TODO: interpolate this test with step count args */ }
+					<p className="create-site__progress-text">{ NO__( 'Step 1 of 3' ) }</p>
```

Comment:
It would be ok to leave this out for the first iteration if it makes merging faster.

Reply:
I've just used good ol' template literals to get the effect. English only :)